colin/checks/result.py

```diff
@@ -28,3 +28,9 @@ class ImageCheckResult(AbstractResult):
     def __init__(self, ok, status, description, message, logs):
         super().__init__(ok, status, description, message)
         self.logs = logs
+
+
+class CheckResults(object):
```

Comment:
What do you mean by this class? What kind of results you would like to check?

Reply:
This class was meant to contain all set of results. (It is a result of whole validation.) Probably, it is not the best name.